harnessed_jobs/og_sweep_raft_acq/v0/ccs_og_sweep_raft_acq.py

```diff
@@ -0,0 +1,93 @@
+"""
+Jython script to run output gate voltage sweep acquisitions at TS8.
+"""
+
+from eo_acquisition import EOAcquisition, PhotodiodeReadout, AcqMetadata, logger
+import numpy as np
+import itertools
+
+class OGSweepAcquisition(EOAcquisition):
+    """
+    EOAcquisition subclass to sweep voltages and take flat image dataset.
+    """
+
+    def __init__(self, seqfile, acq_config_file, metadata, subsystems,
+                 ccd_names, logger=logger):
+
+        super(OGSweepAcquisition, self).__init__(seqfile, acq_config_file,
+                                                 "OGSWEEP", metadata,
+                                                 subsystems, ccd_names,
+                                                 logger=logger)
+
+    def run(self):
+        """
+        Take flat datasets at each voltage setting in accordance to the
+        configuration file.
+
+        Sample config:
+
+        OGSWEEP_WL 500
+
+        ogsweep 1000 2 -3.0 0.0 0.5 
+        """
+
+        openShutter = True
+        actuateXed = False
+        image_type = "FLAT"
+
+        wl = float(self.eo_config.get("OGSWEEP_WL", 500))
+
+        seqno = 0
+        for tokens in self.instructions:
+            target_flux = float(tokens[1])
+            nframes = int(tokens[2])
+            min_vog = float(tokens[3])
+            max_vog = float(tokens[4])
+            step_vog = float(tokens[5])
+            test_type = "OGSWEEP" # Change to desired test name
```

Comment:
You don't need to set this here or below in the call to `take_image` since the default is to use the value you passed to the constructor.